otx/algorithms/action/adapters/openvino/model_wrappers/openvino_models.py

```diff
@@ -0,0 +1,185 @@
+"""Model wrapper file for openvino."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=invalid-name
+
+from typing import Any, Dict, List
+
+import numpy as np
+
+from otx.api.entities.datasets import DatasetItemEntity
+from otx.api.utils.argument_checks import check_input_parameters_type
+
+try:
+    from openvino.model_zoo.model_api.models.model import Model
+    from openvino.model_zoo.model_api.models.utils import (
+        RESIZE_TYPES,
+        Detection,
+        InputTransform,
+    )
+except ImportError as e:
+    import warnings
+
+    warnings.warn(f"{e}, ModelAPI was not found.")
+
+
+@check_input_parameters_type()
+def softmax_numpy(x: np.ndarray):
+    """Softmax numpy."""
+    x = np.exp(x)
```

Comment:
How about `x = np.exp(x - np.max(x))`? Refered #1140

Reply:
> How about `x = np.exp(x - np.max(x))`? Refered #1140

It should use what you suggested to guarantee the numerical stability

Reply:
I agree, Thanks